cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/aws-orchestrator/AwsOrchestrator.service.tsx

```diff
@@ -0,0 +1,33 @@
+// React imports
+import React from 'react';
+import { AwsService } from '../../../../../swagger-api';
+import { AWS_FIELDS } from '../../aws-mc-basic/AwsManagementClusterBasic.constants';
+import { INPUT_CHANGE } from '../../../../../state-management/actions/Form.actions';
+import { AwsDefaults } from '../default-service/AwsDefaults.service';
+import { STORE_SECTION_FORM } from '../../../../../state-management/reducers/Form.reducer';
+import { AWS_ADD_RESOURCES } from '../../../../../state-management/actions/Resources.actions';
+import { AwsResourceAction, FormAction, StoreDispatch } from '../../../../../shared/types/types';
+export class AwsOrchestrator extends React.Component {
+    static async initOsImage(awsState: { [key: string]: any }, awsDispatch: StoreDispatch) {
+        // clear previous os images
```

Comment:
nitpick on style: generally, I find that if I need a comment, it may be helpful to introduce a helper method. So here, what do you think about a one-line helper method called `clearOsImages()`, and then another for `saveOsImages()` and a final one for `setDefaultOsImage()`?

Reply:
Right. That will be much more clear to understand the logic right here. Thanks